Content.Shared/Electrocution/Components/ElectrifiedComponent.cs

```diff
@@ -1,121 +1,125 @@
     public bool OnInteractUsing = true;
 
     /// <summary>
     /// Indicates if the entity requires power to function
     /// </summary>
-    [DataField("requirePower")]
+    [DataField("requirePower"), AutoNetworkedField]
     public bool RequirePower = true;
 
     /// <summary>
     /// Indicates if the entity uses APC power
     /// </summary>
-    [DataField("usesApcPower")]
+    [DataField("usesApcPower"), AutoNetworkedField]
     public bool UsesApcPower = false;
 
     /// <summary>
     /// Identifier for the high voltage node.
     /// </summary>
-    [DataField("highVoltageNode")]
+    [DataField("highVoltageNode"), AutoNetworkedField]
     public string? HighVoltageNode;
 
     /// <summary>
     /// Identifier for the medium voltage node.
     /// </summary>
-    [DataField("mediumVoltageNode")]
+    [DataField("mediumVoltageNode"), AutoNetworkedField]
     public string? MediumVoltageNode;
 
     /// <summary>
     /// Identifier for the low voltage node.
     /// </summary>
-    [DataField("lowVoltageNode")]
+    [DataField("lowVoltageNode"), AutoNetworkedField]
     public string? LowVoltageNode;
 
     /// <summary>
     /// Damage multiplier for HV electrocution
     /// </summary>
-    [DataField]
+    [DataField, AutoNetworkedField]
     public float HighVoltageDamageMultiplier = 3f;
 
     /// <summary>
     /// Shock time multiplier for HV electrocution
     /// </summary>
-    [DataField]
+    [DataField, AutoNetworkedField]
     public float HighVoltageTimeMultiplier = 1.5f;
 
     /// <summary>
     /// Damage multiplier for MV electrocution
     /// </summary>
-    [DataField]
+    [DataField, AutoNetworkedField]
     public float MediumVoltageDamageMultiplier = 2f;
 
     /// <summary>
     /// Shock time multiplier for MV electrocution
     /// </summary>
-    [DataField]
+    [DataField, AutoNetworkedField]
     public float MediumVoltageTimeMultiplier = 1.25f;
 
-    [DataField("shockDamage")]
+    [DataField("shockDamage"), AutoNetworkedField]
     public float ShockDamage = 7.5f;
 
     /// <summary>
     /// Shock time, in seconds.
     /// </summary>
-    [DataField("shockTime")]
+    [DataField("shockTime"), AutoNetworkedField]
     public float ShockTime = 8f;
 
-    [DataField("siemensCoefficient")]
+    [DataField("siemensCoefficient"), AutoNetworkedField]
     public float SiemensCoefficient = 1f;
 
-    [DataField("shockNoises")]
+    [DataField("shockNoises"), AutoNetworkedField]
     public SoundSpecifier ShockNoises = new SoundCollectionSpecifier("sparks");
 
-    [DataField("playSoundOnShock")]
+    [DataField("playSoundOnShock"), AutoNetworkedField]
     public bool PlaySoundOnShock = true;
 
-    [DataField("shockVolume")]
+    [DataField("shockVolume"), AutoNetworkedField]
     public float ShockVolume = 20;
 
-    [DataField]
+    [DataField, AutoNetworkedField]
     public float Probability = 1f;
+
+    [DataField, AutoNetworkedField]
+    public bool IsWireCut = false;
```

Comment:
made this work the same way DoorRemote is checking Bolts - DoorBoltComponent have same property and it is changed in the same way.